telos-frontend/src/components/habit-tracker/HabitTracker.module.css

```diff
@@ -5,12 +5,12 @@
   padding: 0;
 }
 
-.hidden {
-  display: none !important;
-}
+/*.hidden {*/
+/*  display: none !important;*/
+/*}*/
```

Comment:
Is this still needed?

Reply:
Just getting back to this - I forgot I'd commented that instead of deleted it, I'll fix it